doc/enterprise-shield/p_enterprise_shield_config.md

```diff
@@ -0,0 +1,418 @@
+
+1.  [What You Will Accomplish](#step-1-what-you-will-accomplish)
+2.  [Before You Begin](#step-2-before-you-begin)
+3.  [Download and Set Up KAAZING Gateway](#step-3-download-and-set-up-kaazing-gateway)
+4.  [Configure the Internal Gateway](#step-4-configure-the-internal-gateway)
+5.  [Configure the DMZ Gateway](#step-5-configure-the-dmz-gateway)
+6.  [Configure Security Between the DMZ and Internal Gateways](#step-6-configure-security-between-the-dmz-and-internal-gateways)
+7.  [Verify the Gateways are Running](#step-7-verify-the-gateways-are-running)
+8.  [Close Inbound ports](#step-8-close-inbound-ports)
+9.  [Verify the End-to-End Configuration](#step-9-verify-the-end-to-end-configuration)
+
+Step 1: What You Will Accomplish
+---------------------------------------------------------
+
+At the end of this walkthrough, you will have configured a maximum security Enterprise Shield™ topology.
+
+![Use Case 3: Maximum (End-to-End) Transport Security](../images/es_usecase3.png)
+
+**Figure: Maximum (End-to-End) Transport Security**
+
+In this topology, the Gateway on the internal trusted network and a DMZ Gateway collaborate so that inbound ports of the firewall can be closed. Doing so provides a barrier to unauthorized users but still allows full bidirectional access. It works because of the collaboration between the internal Gateway and DMZ Gateway. The internal Gateway is configured to be totally opaque and hides all internal DNS names, IP addresses, and so on. The internal Gateway completes the connection initiated through a client request by accessing the outbound ports on the DMZ Gateway. There are no changes to any application logic.
+
+Step 2: Before You Begin
+----------------------------------------------
+
+This walkthrough covers Use Case 3: Maximum (End-to-End) Transport Security from the set of [Common Enterprise Shield™ Use Cases](c_enterprise_shield_use_cases.md#usecase3) because that is the most secure example of Enterprise Shield™. By the end of this walkthrough, you will have the foundational knowledge required to understand and configure any of the Enterprise Shield™ use cases.
+
+Step 3: Download and Set Up KAAZING Gateway
+---------------------------------------------------------------
+
+Download and set up two KAAZING Gateways, with one Gateway in the DMZ and one in the internal trusted network. This is described in [Setting Up the Gateway](../about/setup-guide.md).
+
+Step 4: Configure the Internal Gateway
+-------------------------------------------------------------------
+
+Perform the following steps to set up the App1 and App2 services on the internal Gateway.
+
+**Notes:**
+-   The examples in this walkthrough use the `.net` domain (such as `tcp://gateway.example.net:8080`) to indicate internal, nonpublic URLs, and use the `.com` domain to indicate public URLs. All domains and URLs are for example purposes only. Wherever a URI uses `example` in the domain name, you should replace that with your own company name.
```

Comment:
> you should replace that with your own company name.

"... you should replace that with your own company name or appropriate hostname."